src/field.jl

```diff
@@ -2,7 +2,7 @@ import Base.show
 
 export CustomParser, Quoted
 
-using Compat
+using Compat, Nullables
```

Comment:
We should get rid of `Nullables`; we should be able to use `Union{Nothing, T}` in place of `Nullable{T}`.